charts/node-red/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.0.6-12
 description: Node-RED is low-code programming for event-driven applications
 name: node-red
-version: 3.0.0
+version: 3.0.1
```

Comment:
As we are introducing a feature, can you please bump the version to 3.1.0?

Reply:
Sure thing